src/Console/ModelsCommand.php

```diff
@@ -637,6 +637,17 @@ public function getPropertiesFromMethods($model)
                                 );
 
                                 if (strpos(get_class($relationObj), 'Many') !== false) {
+                                    if ($relationObj instanceof BelongsToMany) {
+                                        $pivot = get_class($relationObj->newPivot());
+                                        if ($pivot != 'Illuminate\Database\Eloquent\Relations\Pivot') {
```

Comment:
I guess an `instanceof` check isn't necessary here?
```suggestion
                                        if ($pivot !== Illuminate\Database\Eloquent\Relations\Pivot::class) {
```